charter.md

```diff
@@ -1,134 +1,159 @@
+3. Those responsible for accessing data from data mirrors to download to their institution.
+4. Others who wish to observe.
 
 The current membership includes:
 
-Responsibilty | Name | Institution
---- | --- | ---
-Chair | Kevin Krieger | University of Saskatchewan
-Co-Chair | Kevin Sterne | Virginia Tech
-Member | Simon Shepherd | Dartmouth
-Member | Akira Sessai Yukimatu | NIPR
-Member | Kehe Wang | IPS
-Member | Andrew McDonald | LaTrobe
-Member | Paul Breen | BAS
-Member | Member | Bill Bristow | UAF
-Member | Member | Jef Spaleta | UAF
-Member | Jianjun Liu | PRIC
-Member | Judy Stephenson | Durban
-Member | Member | Tomoaki Hori | STELAB
-Member | Member | Nozomu Nishitani | STELAB
-Member | Aurélie Marchaudon | IRAP/CNRS
-Member | Member | Nigel Wade | Leicester
-Member | Member | Tsutomu Nagatsuma | NICT
-Member | Keisuke Hosokawa | DICE
-Member | Ethan Miller | JHU/APL
-Member | Member | Ermanno Amata | IAPS-INAF
-Member | Tom Kane | LaTrobe
-Member | Hong-Qiao Hu | PRIC
-Member | Xavier Vallieres | CNRS
-Member | Chris Thomas | Leicester
-Member | Manabu Kunitake | NICT
-Observer | Mike Ruohoniemi | Virginia Tech
-Observer | Kathryn McWilliams | University of Saskatchewan
-Observer | Dieter Andre | University of Saskatchewan
-Observer | Jean-Pierre St.Maurice | University of Saskatchewan
+Responsibilty | Name | Institution | Radars
+--- | --- | --- | ---
+Chair | Kevin Krieger | USASK | sas, pgr, rkn, inv, cly
+Member | Marci Detwiller | USASK | sas, pgr, rkn, inv, cly
+Member | Marina Schmidt | USASK | sas, pgr, rkn, inv, cly
+Member | Keith Kotyk | USASK | sas, pgr, rkn, inv, cly
+Member | Kevin Sterne | VT | bks, kap, gbr, fhe, fhw
+Member | Simon Shepherd | Dartmouth | cve, cvw
+Member | Evan Thomas | Dartmouth | cve, cvw
+Member | Akira Sessai Yukimatu | NIPR | sye, sys
+Member | Jordan Wiker | JHU/APL | wal
+Member | Enrico Simeoli | IAPS/INAF | dce, dcn
+Member | Kehe Wang | BOM
+Member | John Devlin | LaTrobe | tig, unw, bpk
+Member | Jim Whittington | LaTrobe | tig, unw, bpk
+Member | Mark Gentile | LaTrobe | tig, unw, bpk
+Member | Andrew McDonald | LaTrobe | tig, unw, bpk
+Member | Paul Breen | BAS | fir
+Member | Tim Barnes | BAS | fir
+Member | Bill Bristow | UAF/Penn State | sps, mcm, ade, adw, kod
+Member | Jianjun Liu | PRIC | zho
+Member | Hongqiao Hu | PRIC | zho
+Member | Judy Stephenson | UKZN | san
+Member | Tomoaki Hori | ISEE | hok, hkw
+Member | Nozomu Nishitani | ISEE | hok, hkw
+Member | Aurélie Marchaudon | IRAP | ker
+Member | Tsutomu Nagatsuma | NICT | ksr
+Member | Keisuke Hosokawa | DICE/UEC |
+Member | Chris Thomas | Leicester | han, pyk, sto
+Member | Tim Yeoman | Leicester | han, pyk, sto
+Member | Qinghe Zhang | SDU
+Member | Zanyang Xing | SDU
+Member | Mikko Syrjäsuo| UNIS | lyr
+Observer | Mike Ruohoniemi | VT
+Observer | Kathryn McWilliams | USASK
+Observer | Dieter Andre | USASK
+Observer | Jean-Pierre St.Maurice | USASK
+Observer | Federica Marcucci | IAPS/INAF 
```

Comment:
Missing 
- Lancaster 
- JME (missing above in institutions)

Reply:
Emailed Jiaojaio about JME and Jim about sto